experimenter/tests/integration/nimbus/test_create_mobile_experiment.py

```diff
@@ -5,21 +5,25 @@
 
 import pytest
 import requests
-from nimbus.models.base_dataclass import BaseExperimentApplications
+
 from nimbus.pages.experimenter.summary import SummaryPage
 from nimbus.utils import helpers
 
 
 @pytest.fixture
-def experiment_slug():
-    return "firefox-ios-integration-test"
+def experiment_slug(application):
+    return f"firefox-{application.lower()}-integration-test-experiment"
 
 
-@pytest.fixture
-def default_data_api(default_data_api):
-    feature_config_id = helpers.get_feature_id_as_string(
-        "messaging", BaseExperimentApplications.FIREFOX_IOS.value
-    )
+def test_create_mobile_experiment_for_integration_test(
+    selenium, experiment_url, kinto_client, default_data_api, experiment_slug, application
+):
+    """Create a mobile experiment for device integration tests"""
+    apps = ["IOS", "FENIX"]
```

Comment:
Should this be a 'params' fixture here?

Reply:
No this is used for the `pytest.skip()` in the following lines.